lua/lspconfig/server_configurations/dagger.lua

```diff
@@ -0,0 +1,22 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'cuelsp' },
```

Comment:
Consider stating in the description that one can use
```lua
cmd = { 'dagger', 'cuelsp' },
```
here if they have only installed `dagger` and not both `dagger` and `cuelsp`.

Reply:
the idea behind this PR is that it could be potentiall used with `https://github.com/williamboman/nvim-lsp-installer`. And since this LSP works also with `only` CUE files, I didn't want people to unnecessarily install dagger for that. It might be better to just use `cuelsp` so it can be independently updated from Dagger

Reply:
Note that nvim-lsp installer is deprecated, and mason https://github.com/williamboman/mason.nvim is the new thing. I would recommend submitting cuelsp package there.